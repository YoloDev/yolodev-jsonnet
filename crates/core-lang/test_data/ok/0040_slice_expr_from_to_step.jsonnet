local foo = [0];

foo[0:10:2]
