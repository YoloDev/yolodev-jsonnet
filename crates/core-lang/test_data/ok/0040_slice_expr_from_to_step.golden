local foo#2 = [0.0];
std#1["slice"](foo#2, 0.0, 10.0, 2.0)
